pycbc/types/timeseries.py

```diff
@@ -856,19 +856,39 @@ def add_into(self, other):
         other = other.copy()
         dt = float((other.start_time - self.start_time) * self.sample_rate)
 
+        # This coaligns other to the time stepping of self
         if not dt.is_integer():
             diff = (dt - _numpy.floor(dt)) * self.delta_t
```

Comment:
Might be cleaner to use `divmod` here so that you get the integer and fractional parts at the same time.

Reply:
I'd prefer to keep as is.